packages/fluentui/react-northstar/src/components/Tree/Tree.tsx

```diff
@@ -161,7 +160,16 @@ const iterateItems = (items: TreeProps['items'] | TreeItemProps['items'], acc =
     acc,
   );
 
-const Tree: React.FC<WithAsProp<TreeProps>> &
+/**
+ * A Tree displays data organised in tree hierarchy.
+ *
+ * @accessibility
+ * Implements [ARIA TreeView](https://www.w3.org/TR/wai-aria-practices-1.1/#TreeView) design pattern.
+ * @accessibilityIssues
+ * [Treeview - JAWS doesn't narrate position for each tree item](https://github.com/FreedomScientific/VFO-standards-support/issues/338)
+ * [Aria compliant trees are read as empty tables](https://bugs.chromium.org/p/chromium/issues/detail?id=1048770)
+ */
+const Tree: ComponentWithAs<'div', TreeProps> &
```

Comment:
The change from `ul` to `div` is intentional bugfix, correct?

Reply:
Yep, as `Tree` and its subcomponents render `div`s:

![image](https://user-images.githubusercontent.com/14183168/86350076-6f661780-bc62-11ea-8b4f-749a4e1dbe6b.png)

I also double checked this with @jurokapsiar and we will avoid `ul`/`li` as they are breaking markup once you add custom scrollbar or virtualization